pallets/posts/src/functions.rs

```diff
@@ -340,4 +340,157 @@ impl<T: Config> Pallet<T> {
 
         Ok(())
     }
+
+    pub fn ensure_account_can_create_post(account: T::AccountId, space_id: SpaceId) -> DispatchResult {
```

Comment:
This function never used

<img width="341" alt="image" src="https://github.com/dappforce/subsocial-parachain/assets/39906111/9103f284-903d-4958-b767-803cac8e4ef1">

Reply:
Looks like it just outdated method, because you implemented almost same in check_account_can_create_post